README.md

```diff
@@ -9,6 +9,12 @@ Scripts to create in silico lipid fragmentation spectra, based on analysis of ch
 
 3. `python setup.py sdist bdist_wheel`
 
+# Run Tests
+
+1. `pip install pytest`
+
+2. `pytest test.py`
+
```

Comment:
Should there be a `test.py` file added to the repo?
Otherwise lgtm

Reply:
Yes just cleaned that up

Reply:
👍 lgtm